app/src/main/java/mozilla/lockbox/adapter/SettingListAdapter.kt

```diff
@@ -51,7 +53,7 @@ class SettingListAdapter : RecyclerView.Adapter<SettingViewHolder>() {
     }
 
     override fun onBindViewHolder(holder: SettingViewHolder, position: Int) {
-        holder.compositeDisposable.clear()
+        holder.disposable?.let { compositeDisposable.remove(holder.disposable!!) }
```

Comment:
This looks needlessly complicated. Also avoid `!!`.

Would work: 
```kt
 holder.disposable?.let { compositeDisposable.remove(it) }
```

Reply:
👍